webapps/frontend/camunda-commons-ui/lib/widgets/inline-field/cam-widget-inline-field.html

```diff
@@ -84,7 +84,7 @@
   <span ng-show="varType !== 'option'"
         class="cam-widget-inline-field btn-group">
     <button type="button"
-                ng-if="inOperator && !(varType === 'datetime' || varType === 'date' || varType === 'time')"
+                ng-if="(inOperator || notInOperator) && !(varType === 'datetime' || varType === 'date' || varType === 'time')"
```

Comment:
We had discussion that floated the idea to not use Operator `Key` to show expanded edit button, instead add a new property. This would mean that we would have to add the new key to all places where `In` or `NotIn` value should be expanded. 
I checked again and the only other places where the notInOperator is used: 
- in `Search case instances` the caseDefinitionKey `NotIn` filter, which is not working as it is, created a issue to fix it: [Cockpit - fix caseDefinitionKeyNotIn filter in case search](https://github.com/camunda/camunda-bpm-platform/issues/4932)
- in `batch/operation` page `PIprocessDefinitionKey` has a `NotIn` filter option. It send an array type value as filter value so showing the expand button here is correct. 

I think adding check on `notInOperator` is the way to go.